tests/cli.rs

```diff
@@ -289,6 +289,31 @@ fn import_multiple_no_tags() -> color_eyre::Result<()> {
     Ok(())
 }
 
+// This test triggers Github rate limit when ran by CI.
+// Making it target a single platform as a hacky workaround.
```

Comment:
This is fine, better than no testing!

Reply:
Looks like it's still failing. You can put an #[ignore] tag to this test so CI doesn't run it.